lib/backend/pgbk/pgbk.go

```diff
@@ -92,6 +108,15 @@ type Config struct {
 	DisableExpiry   bool           `json:"disable_expiry"`
 	ExpiryInterval  types.Duration `json:"expiry_interval"`
 	ExpiryBatchSize int            `json:"expiry_batch_size"`
+
+	// DisableChangeFeed prevents the default postgres changefeed from running
+	// This is helpful for postgres like backends to manage their own change feed.
+	DisableChangeFeed bool `json:"-"`
+	// DisableSetupAndMigration prevents schema creation/migration from running.
+	// This allows postgres like backends to manage their own schema.
+	DisableSetupAndMigration bool `json:"-"`
+	// Pool allows passing in a pgxpool rather than the backend creating one itself.
+	Pool *pgxpool.Pool `json:"-"`
```

Comment:
I'm not at all convinced that this will lead to a good design. I am concerned that this will essentially prevent any decent work in pgbk because any change will also need to be fully compatible with other backends that are piggybacking on the exact details of the current implementation.